tasks/aludel.ts

```diff
@@ -198,48 +197,37 @@ task("delist-program")
     await (await factory.delistProgram(args.program)).wait();
   });
 
-task("add-program")
-  .addParam("aludelFactory", "address of the aludel factory")
```

Comment:
We could use these tasks for locally deployed contracts, and it would be helpful to overwrite deployed addresses. 

I think we need to leave it as an optional param